paymentsheet/src/main/java/com/stripe/android/paymentsheet/utils/SelectionUtils.kt

```diff
@@ -0,0 +1,21 @@
+@file:RestrictTo(RestrictTo.Scope.LIBRARY)
+
+package com.stripe.android.paymentsheet.utils
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.paymentsheet.PaymentSheet
+import com.stripe.android.paymentsheet.model.PaymentSelection
+
+internal fun PaymentSelection.New.canSave(
```

Comment:
I don't see anything here about there being a customer attached. Is that handled elsewhere?

Reply:
At the moment, that is handled internally by `DefaultPrefsRepository` which does save guest preferences. 

```
// DefaultPrefsRepository.kt
private fun getKey(): String {
    return customerId?.let { "customer[$it]" } ?: "guest"
}
```

Not sure if there a reason we want to save guest preferences though. If not, we could can logic in `canSave` that checks for a customer id.